libkbfs/md_journal.go

```diff
@@ -1178,7 +1226,11 @@ func (j *mdJournal) clear(
 	// Garbage-collect the old branch entries.  TODO: we'll eventually
 	// need a sweeper to clean up entries left behind if we crash
 	// here.
+	isLocalSquash := bid == LocalSquashBranchID
```

Comment:
Ah, ok, I see, where it's called below, it's called on `otherJournal` so we don't actually care if all its entries are nuked. I'm still inclined to have this function behave 'correctly', though. But if not, definitely need a comment explaining the behavior.